poster/poster.tex

```diff
@@ -126,7 +126,7 @@
 Identify flexible, general, and performant algorithms available for application to simulating
 demand-driven deployment of nuclear fuel cycle facility capacity in a fuel cycle simulator.
 \begin{itemize}
-	\item Review nuclear fuel cycle simulator state-of-the-art.
+	\item Review state-of-the-art nuclear fuel cycle simulators.
```

Comment:
my point was that we were reviewing "what is the nuclear fuel cycle simulator state-of-the-art". That is, the goal was to determine the boundary of nuclear fuel cycle simulator capabilities. I think this second wording says we only looked at the best simulators.  I can see how my phrasing is confusing, but this second phrasing changes the meaning. Perhaps  "Review existing nuclear fuel cycle simulator capabilities." is clearest.

Reply:
I see! That makes a lot of sense. As long as the meaning is preserved, I'll trust your judgement on that.